cmd/api/adapter/invitation/handler.go

```diff
@@ -19,8 +19,8 @@ type handler struct {
 	usecase invitation.Usecase
 }
 
-func NewHandler(repo registry.Repository, srvc registry.Service) handler {
-	usecase := invitation.NewUsecase(repo, srvc)
+func NewHandler(repo registry.Repository) handler {
```

Comment:
🚫 **[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported func NewHandler returns unexported type github.com/ispec-inc/go-distributed-monolith/cmd/api/adapter/invitation.handler, which can be annoying to use (golint)